testing/xfpga/test_enum_c.cpp

```diff
@@ -84,8 +84,9 @@ class enum_c_p : public ::testing::TestWithParam<std::string> {
     }
 
     int value;
-    std::string cmd = "ls -l /sys/class/fpga/intel-fpga-dev* | "
-                      "wc -l";
+    std::string cmd =
+        "(ls -l /sys/class/fpga*/region*/*fme*/dev || "
+        "/sys/class/fpga*/*intel*) |  (wc -l)";
```

Comment:
Isn't there supposed to be another ls in front of /sys/class/fpga*/\*intel*?

Reply:
fixed